src/main/java/org/junit/runner/JUnitCommandLineParseResult.java

```diff
@@ -85,13 +85,11 @@ private void parseArgs(String[] args) {
     }
 
     private String[] copyArray(String[] args, int from, int to) {
-        ArrayList<String> result = new ArrayList<String>();
-
+        String [] result = new String[to-from];
         for (int j = from; j != to; ++j) {
-            result.add(args[j]);
+            result [j-from] = args[j];

```

Comment:
We don't put spaces when indexing into an array. Please change to:

    result[j - from] = args[j];